### DISCUSSION TITLE
Logo for PyCon India 2024

### DISCUSSION THREAD

**Discussion by anistark** [pythonindia/inpycon2024-tasks#27] [General]
We're finalising the logo for PyCon India 2024.

Here's selected options we have received, and improvised upon. Please feel free to share your thoughts. We will conclude the discussion on March 9th, 2024. 

The team is coordinating with the T-shirt vendors on which typography fits the best. Using all the inputs, the team will decide the typography in the week of March 11th:

1. 
![t1](https://github.com/pythonindia/inpycon2024-tasks/assets/5357586/837908ee-c4b6-48d4-8820-d073ca8e57eb)

2. 
![inpycon-c](https://github.com/pythonindia/inpycon2024-tasks/assets/5357586/8f98aeb4-f946-43e2-8d9e-0d209a1f1b35)

3. 
![t2](https://github.com/pythonindia/inpycon2024-tasks/assets/5357586/2015ac3c-d80d-449f-8c6c-2e2115d25f25)

**Comment by satyaakam** [comment]
Option 3 , as per this documentation https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#creating-a-poll , we can create a poll for these kinds of discussions

**Comment by ghoseb** [comment]
Option #1. Better contrast and clear, modern typography.

**Reply by SaptakS** [reply]
Agreed.

**Reply by forageaiadmin** [reply]
Option #1

**Comment by s-jaiprakash** [comment]
Option 3

**Reply by kyogesh** [reply]
Option 3

**Comment by scari** [comment]
First of all, I'd like to confess that I am a huge fan of all the logos from IN team. 😉
All three options are great, but if I had to choose one, I'd go with number 2. I feel the logo calmer and more organized.
🙏

**Comment by aroraumang** [comment]
I like Option 2 🫶

**Comment by mohitsuman** [comment]
I would put my vote to design no 2. 

The Bengaluru and 2024 position provides a different viewpoint and would directly capture the user to the location of PyCon.

**Comment by SanchitBahl** [comment]
Option 2 seems to be the most pleasing to (at least) my eyes.

It's as clean as it could have been. Whereas option 1 is not much different other than the contrast. Option 3 feels indigenous but a bit cluttered to be used as a logo.
After all, all of them are amazing logos. 🙌🏻

**Comment by navyad** [comment]
Optioins 1. with clear presentation

**Comment by ChaitanyaBhole** [comment]
Option 3 is better.

**Comment by nerdyk3** [comment]
Option 2

**Comment by kabir4691** [comment]
Option 3.

**Comment by shrayasr** [comment]
Option 1.

Neat and crisp.

**Comment by anistark** [comment]
On all suggestions and further discussions, we've settled for a combination of 1 & 2 color and concept. 

![PyCon_2024](https://github.com/pythonindia/inpycon2024-tasks/assets/5357586/0b8c2ddb-07dc-46d2-89d0-62eb66bc4a4f)

🎉
Thanks everyone for your valuable feedback!
